webapp/src/widgets/graph/Graph.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Line, defaults } from 'react-chartjs-2'
+
+defaults.global.defaultFontSize = '8'
```

Comment:
Why is this necessary? And what it does?

Reply:
it is used to set labels fontsize